app/javascript/gobierto_dashboards/webapp/containers/contract/ContractsShow.vue

```diff
@@ -1,18 +1,18 @@
 <template>
   <div>
-    <h1>{{ formattedContract.title }}</h1>
+    <h1>{{ contract.title }}</h1>
```

Comment:
I notice you don't use `contract` object in any way. What if you declare each variable (title, description, ...) in the data() and only work with its properties?

Reply:
Will do!

Reply:
@Crashillo now I have this:

```javascript 
    this.title = contract.title
    this.description = contract.description
    this.assignee = contract.assignee
    this.document_number = contract.document_number
    this.final_amount = contract.final_amount
    this.initial_amount = contract.initial_amount
    this.status = contract.status
    this.process_type = contract.process_type
```

Is there a way to do it with destructuring?

Reply:
Two steps:
```
    const {
      title,
      description,
      assignee,
      document_number,
      final_amount,
      status,
      process_type
    } = contract

this.title = title
....
```

One step (perhaps more confusing)
```
    ({
      title: this.title,
      description: this.description,
      assignee: this.assignee,
      document_number: this.document_number,
      final_amount: this.final_amount,
      initial_amount: this.initial_amount
      status: this.status,
      process_type: this.process_type
    } = contract)
```

You might want to initialize them also.

Reply:
Thanks!